radixdlt-core/radixdlt/src/main/java/com/radixdlt/middleware2/network/GetVerticesResponseMessage.java

```diff
@@ -84,17 +84,15 @@
 	@DsonOutput(Output.ALL)
 	private final List<UnverifiedVertex> vertices;
 
-	GetVerticesResponseMessage() {
-		// Serializer only
-		this.vertices = null;
-	}
-
-	GetVerticesResponseMessage(List<UnverifiedVertex> vertices) {
+	@JsonCreator
+	public GetVerticesResponseMessage(
+		@JsonProperty(value = "vertices", required = true) List<UnverifiedVertex> vertices
```

Comment:
We aren't checking for nulls inside the list.

Reply:
Depends on whether nulls can be injected by our serializer (aka Cbor).

Reply:
But we can't assume we will receive only message serialised but the system, can we?

Reply:
Ah I think you mean the deserialisation process won't create a List with null inside? But this isn't the only way of creating the object and allowing this could cause unexpected behaviour in the system.